src/deroff.rs

```diff
@@ -1,10 +1,40 @@
 /// A translation of https://github.com/fish-shell/fish-shell/blob/e7bfd1d71ca54df726a4f1ea14bd6b0957b75752/share/tools/deroff.py
 // """ Deroff.py, ported to Python from the venerable deroff.c """
+extern crate regex;
```

Comment:
Don't need to declare crate when using 2018 edition.

Reply:
Done in 5e03a24 :-)